Gemfile

```diff
@@ -28,6 +28,12 @@ gem 'coffee-rails', '~> 4.0.0'
 # See https://github.com/sstephenson/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
 
+# Pusher gem
+gem 'pusher'
+
+# Travis gem
+gem 'travis'
```

Comment:
I think this could be in groups `:development, :test` only. Don't need to load in production